themes/qgrid/layout/partials/nav.ejs

```diff
@@ -24,20 +24,25 @@
                     pages.sort(function (a, b) {
                         return a.order - b.order;
                     });
-
+                    var pattern = new RegExp(/^#+.*/, 'mg');
                     for (var j = 0; j < pages.length; j++) {
```

Comment:
var tags = (p.raw.match(pattern) || []).map(function(x) { return x.replace... }

be more functional please ;)